radiomics/imageoperations.py

```diff
@@ -628,13 +628,35 @@ def getOriginalImage(inputImage, **kwargs):
 
 
 def getLoGImage(inputImage, **kwargs):
-  """
+  r"""
   Applies a Laplacian of Gaussian filter to the input image and yields a derived image for each sigma value specified.
 
+  A Laplacian of Gaussian image is obtained by convolving the image with the second derivative (Laplacian) of a Gaussian
+  kernel.
+
+  The Gaussian kernel is used to smooth the image and is defined as
+
+  .. math::
+
+    G(x, y, z, \sigma) = \frac{1}{(\sigma \sqrt{2 \pi})^3}e^{-\frac{x^2 + y^2 + z^2}{2\sigma^2}}
+
+  The Gaussian kernel is convolved by the laplacian kernel :math:`\nabla^2G(x, y, z)`, which is sensitive to areas with
+  rapidly changing intensities, enhancing edges. The width of the filter in the Gaussian kernel is determined by
+  :math:`\sigma` and can be used to emphasize more fine (low :math:`\sigma` values) or coarse (high :math:`\sigma`
+  values) textures.
+
+  .. warning::
+
+    The LoG filter implemented in PyRadiomics is a 3D LoG filter, and therefore requires 3D input. Features using a
```

Comment:
I would also include a reference to the ITK class that implements the filter for completeness: https://itk.org/Doxygen/html/classitk_1_1LaplacianRecursiveGaussianImageFilter.html

Reply:
I [added](https://github.com/Radiomics/pyradiomics/pull/345/commits/00408d0a9f84968b9b9a52ac2f09ca7358d32fcb#diff-0fc4a9c8cbef7d416192056c5f0632f3R671 the reference to ITK in addition to the reference to SimpleITK